packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -120,13 +120,9 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
     messages.add(new ValidationMessage('Android SDK at ${androidSdk.directory}'));
 
     messages.add(new ValidationMessage(androidSdk.ndkDirectory == null
-          ? 'Unable to locate Android NDK.\n'
+          ? 'Android NDK location unknown (optional, useful for using native profiling tools)'
```

Comment:
Perhaps, `Android NDK location not configured (this optional configuration is useful for native profiling support)`?

Reply:
Done